src/main/webapp/js/user-page-loader.js

```diff
@@ -94,9 +94,9 @@ function buildMessageDiv(message) {
 /** Fetches data and populates the UI of the page. */
 function buildUI() {
```

Comment:
Why is this function changed?
Why is the order of the calls different? Is it intentional?